remix/app/routes/_app.newsletter.create.tsx

```diff
@@ -26,31 +29,43 @@ export async function action({ request }: ActionArgs) {
       ...result.data,
       userId: user.id,
     };
-
     const newsletter = await NewsletterRepository.create({
       data: newsletterData,
     });
 
-    return json({ ...newsletter });
+    session.flash(
+      'toastMessage',
+      `${newsletter.name} Newsletter created successfully`
+    );
+    session.flash('toastMode', 'success');
+
+    return redirect(`/`, {
+      headers: { 'Set-Cookie': await commitSession(session) },
+    });
   });
 }
 
 export default function Index() {
   return (
-    <>
-      <h1>Create Newsletter page </h1>
-      <Form method="POST">
-        <div>
-          <label htmlFor="name">Name</label>
-          <input type="text" name="name" placeholder="name" />
-        </div>
-        <br />
-        <div>
-          <label htmlFor="content">Content</label>
-          <textarea name="content" placeholder="content" />
+    <section className="flex flex-col gap-4">
+      <header className="text-xl font-bold">
+        <h1>Create your Newsletter </h1>
+      </header>
+      <ValidatedForm
+        validator={validator}
+        method="POST"
```

Comment:
Does this need to be capitalized? It is not in the `AuthSignInPage` route 🤔 So it looks inconsistent

Reply:
Oh, that is one of my mistakes too, As a newer version of HTML we should use `lowercase` instead of `uppercase`

I will fix it.

 [Reference](https://stackoverflow.com/questions/7790576/what-are-http-method-capitalization-best-practices).

Reply:
Fixed here: [08ab23a](https://github.com/nimblehq/js-framework-benchmark/pull/120/commits/08ab23adcb5bda8ceb549edbbc9a3671d2288375)